index.html

```diff
@@ -946,14 +864,143 @@ <h4>publicKeyBase58</h4>
                 <a href="https://w3id.org/security/#publicKeyBase58">security-vocab</a>
               </td>
               <td>
-                <a href="https://www.w3.org/ns/did/v1">DID Core context</a>
+                <a href="https://www.w3.org/ns/did/v1">DID Core</a>
               </td>
               <td>
                 <a href="cddl/publicKeyBase58.cddl" target="_blank">publicKeyBase58.cddl</a>
               </td>
             </tr>
           </tbody>
         </table>
+      </section>
+
+        <section>
+          <h4>publicKeyHex</h4>
+          <table class="simple" style="width: 100%;">
+            <thead>
+              <tr>
+                <th>Normative Definition</th>
+                <th>JSON-LD</th>
+                <th>CDDL</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <a href="https://w3id.org/security#publicKeyHex">security-vocab</a>
+                </td>
+                <td>
+                  <a href="https://w3id.org/security/v3-unstable">security-vocab/v3-unstable</a>
+                </td>
+                <td>
+                </td>
+                <td>
+                  <a href="cddl/publicKeyHex.cddl" target="_blank">publicKeyHex.cddl</a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <pre class="example" title="Example of publicKeyHex property">
+  {
+    "@context":[
+      "https://www.w3.org/ns/did/v1",
+      "https://identity.foundation/EcdsaSecp256k1RecoverySignature2020#"
+    ],
+    "id":"did:example:123",
+    "publicKey":[{
+      "id": "did:example:123#vm-2",
+      "controller": "did:example:123",
+      "type": "EcdsaSecp256k1RecoveryMethod2020",
+      "publicKeyHex": "027560af3387d375e3342a6968179ef3c6d04f5d33b2b611cf326d4708badd7770"
+    }]
+  }
+          </pre>
+        </section>
+
+        <section>
+          <h4>blockchainAccountId</h4>
+          <div class="issue" data-number="165"></div>
+          <table class="simple" style="width: 100%;">
+            <thead>
+              <tr>
+                <th>Normative Definition</th>
+                <th>JSON-LD</th>
+                <th>CDDL</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <a href="https://w3id.org/security#blockchainAccountId">security-vocab</a>
+                </td>
+                <td>
+                  <a href="https://www.w3.org/ns/did/v1">DID Core</a>
```

Comment:
```suggestion
                  <a href="ttps://w3id.org/security/v3-unstable">security-vocab v3</a>
```

Reply:
@rhiaro without this, there will not be a single did core verification method compatible with blockchain addresses. many folks in the community spent months trying to get `ethereumAddress` or `bitcoinAddress` added to did core context, so that "no extensions" would be required to create a DID that uses a public blockchain.... we should talk about this, if I need to open a PR to DID Core to add an example of blockchainAccountId, next to to publicKeyBase58, I will :)

Reply:
> we should talk about this, if I need to open a PR to DID Core to add an example of blockchainAccountId, next to to publicKeyBase58, I will :)

Yeah, we may need to do this. It's still not clear where the line is wrt. what goes in Core DID v1 context and what doesn't... I think the PR you raised rewriting the verification method section makes this even more clear (that it's an issue). For example, what key types are we defining in DID Core? Ed25519? Bitcoin/Ethereum? JOSE? BLS? All of them, some of them, none of them -- shouldn't what is supported be a DID Method decision? 

The decision is a bit easier wrt. blockchainAccountId vs. BlsVerificationKey2020... what we did in the VC Data Model spec seems to be holding... but we all know we're going to need a v2 context there sooner than later.

In any case, agreed that the LD folks need to get together to figure out how we're going to provide guidance wrt. what goes in DID Core context and what goes in DID Method specific context.

Reply:
There are open PRs on the VC Data Model, which make it clear that how it was handled there is not preferred by everyone: https://github.com/w3c/vc-data-model/pull/759

^ this is a perfect example of how little consensus we have on this topic.